tests/integration/fixtures/processing.py

```diff
@@ -302,7 +313,7 @@ def assert_rate_limited(
         key_id=None,
         categories=None,
         quantity=None,
-        timeout=1,
+        timeout=None,
```

Comment:
Drive-by: `assert_rate_limited` did not use a shorter timeout than the default one so was prone to flake.